src/transformers/generation/logits_process.py

```diff
@@ -1702,3 +1702,28 @@ def __call__(self, input_ids, scores):
         unconditional_logits = torch.nn.functional.log_softmax(logits[:, -1], dim=-1)
         out = self.guidance_scale * (scores - unconditional_logits) + unconditional_logits
         return out
+
+
+class BarkEarlyStoppingLogitsProcessor(LogitsProcessor):
```

Comment:
We could probably remove `Bark` mention here since it could be useful for other models here.
Let's ask for @gante opinion here!